src/client/components/pages/entities/annotation.js

```diff
@@ -25,12 +25,26 @@ import React from 'react';
 class EntityAnnotation extends React.Component {
 	constructor(props) {
 	  super(props);
-
+	  this.preRef = React.createRef();
 	  this.state = {
-			open: false
+			open: false,
+			show: true
 	  };
 	}
 
+
+	componentDidMount() {
+		const {annotation} = this.props.entity;
+		if (!annotation || !annotation.content) {
+			return;
+		}
+		const spanElement = document.querySelector('pre span');
```

Comment:
It would probably be better to target the element with more specificity, like using the `.annotation-content` class.
Maybe this?
```suggestion
		const spanElement = document.querySelector('.annotation-content span');
```